lib/aws_auditor/rds_instance.rb

```diff
@@ -23,18 +24,32 @@ def initialize(rds_instance)
         self.instance_type = rds_instance.db_instance_class
         self.engine = rds_instance.engine
         self.count = 1
+
+        if tag_name
+          region = rds_instance.availability_zone.split(//).first(9).join
```

Comment:
Consider making a shared method to parse region from zone since you use this in both RDS and Cache.